app/services/data/get-individual-overview.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (id) {
            'workload_points_calculations.available_points AS availablePoints',
            'workload_points_calculations.total_points AS totalPoints',
            'workload.total_cases AS cases',
-           'workload_points_calculations.contracted_hours AS contractedHours',
+           'workload_owner.contracted_hours AS contractedHours',
```

Comment:
Not sure what your standard approach for this is , but could do this in the view using a COALESCE.  Then application doesn't need to worry, but means pushing logic into the DB, which may not be ideal.